linux/just_git_airgap_repo.bsh

```diff
@@ -1326,52 +1338,83 @@ function git_submodule_sync()
 {
   local remote_name="${1-origin}"
 
-  local JUST_GIT_UPSTREAM="${JUST_GIT_UPSTREAM-just_upstream}"
-
   local OLD_IFS="${IFS}"
   IFS=$'\n'
```

Comment:
If you make `IFS` local here, you wouldn't need to worry about resetting IFS before every `return`